lib/webengine.cpp

```diff
@@ -17,26 +17,61 @@
 Q_GLOBAL_STATIC(SailfishOS::WebEngine, webEngineInstance)
 
 /*!
-  \class SailfishOS::WebEngine
-  \brief Provides access to the Web engine context
-  \inmodule SailfishWebView
+    \class SailfishOS::WebEngine
+    \brief Provides access to the web engine context.
+    \inmodule SailfishWebView
+    \inherits QMozContext
 
-  Singleton class which provides access to the Web engine context.
+    Singleton class which provides access to the web engine context.
 */
 
 namespace SailfishOS {
 
 /*!
-  \brief Initialises the WebEngine class.
+    \brief Initialises the WebEngine class.
 
-  Initalises the WebEngine class. The \a profilePath sets the root folder for
-  the mozilla gecko profile. Once set the \a profilePath can't be changed.
+    Initalizes the WebEngine class. The \a profilePath sets the root folder for
+    the mozilla gecko profile. Once set the \a profilePath can't be changed.
+    Set \a runEmbedding to \c false (defaults to true) in order to delay
+    execution of the EmbedLite event loop. In this case, \l runEmbedding must
```

Comment:
Something like:
```
in order to handle steps that are needed before \l{engine startup}{runEmbedding}.
``` 
Point is that that let's try to avoid "EmbedLite event loop".  I think after the change "In this case, \ runEmbedding..." can be removed altogether.

Reply:
Thanks; I've updated this with your suggestion; it looks clear to me.